template/template.go

```diff
@@ -205,24 +205,25 @@ func (kv KV) Values() []string {
 // as this will confuse them and prevent simple things like
 // simple equality checks to fail. Map everything to float64/string.
 type Data struct {
```

Comment:
"Few" is a relative term. It's not hard to imagine a situation where you'd have thousands of alerts per second worth of webhook firing.